docs/ARCHITECTURE.md

```diff
@@ -19,6 +19,13 @@ The following examples indicate how this SDK could eventually be used.
 
 #### Generate text using any suitable model from any provider (most basic example)
 
+##### Fluent API
+```php
+$text = AiClient::prompt('Write a 2-verse poem about PHP.')
+    ->generateText();
```

Comment:
```suggestion
$text = AiClient::prompt('Write a 2-verse poem about PHP.')->generateText();
```
I'm tempted to only break the chaining if we don't immediately terminate the builder. A single line just shows how awesome simple it is to do a basic prompt. Not a big deal either way, just a thought. 😄

Reply:
IMO keeping separate lines is better for readability.